backend/api/src/api/v1/gql/mod.rs

```diff
@@ -77,6 +79,17 @@ impl Subscription {
     async fn noop(&self) -> Result<impl Stream<Item = bool>> {
         Ok(futures_util::stream::iter(Vec::new()))
     }
+
+    async fn new_message(
+        &self,
+        ctx: &Context<'_>,
+        chat_id: i64,
+    ) -> Result<impl Stream<Item = models::message::Message>> {
+        Ok(chat::ChatSubscription
+            .new_message(ctx, chat_id)
+            .await
+            .unwrap())
+    }
```

Comment:
It seems you have this method duplicated why not add it the same way as a field on the subscription object?

Reply:
I had to use `MergedSubscription` I couldn't get it to work another way